### ISSUE TITLE
Deleting a review does not update the number of reviews for the add-on

### ISSUE THREAD

**Issue by bobsilverberg** [mozilla/addons-frontend#6105]
~~Depends on #6084~~

STR:
1. Go to an add-on's detail page.
2. Create a review for the add-on.
3. Note the number of reviews the add-on has (in the upper right-hand card).
4. Click on the link to view all reviews for the add-on.
5. Observe the number of reviews shown in the header for the review listing. It should be (and is) the same number as observed in step 3.
6. Delete your review.
7. Observe the number of reviews shown in the header for the review listing. It should be decremented by 1.
8. Click on the add-on's name in the left-hand card to navigate back to the details for the add-on.
9. Observe the number of reviews the add-on has (in the upper right-hand card). It should also have been decremented by 1.

Expected:
The number of reviews in steps 7 and 9 should be 1 less than the number from step 3.

Actual:
The number of reviews in steps 7 and 9 are the same as the number from step 3.

**Comment by kumar303** [comment]
Does it fix itself after a refresh? This looks similar to https://github.com/mozilla/addons-frontend/issues/5986

Note that dev is taking 7 minutes to show updated rating counts right now. https://github.com/mozilla/addons-server/issues/9293

**Comment by bobsilverberg** [comment]
> Does it fix itself after a refresh?

It does. I'm pretty sure it's a caching issue, rather than a timing issue.

**Comment by ioanarusiczki** [comment]
@bobsilverberg - I tested it on AMO dev with FF62.
In steps 7 and 9, number of reviews is 1 less than the number from step 3. 
But to see the right number of reviews the page must be refreshed after submitting the review. - Could be because of https://github.com/mozilla/addons-server/issues/9293  ?

![on amo dev](https://user-images.githubusercontent.com/33448286/45295985-2f6ff680-b509-11e8-8e3b-a775575a2838.gif)

**Comment by bobsilverberg** [comment]
@ioanarusiczki Thanks for taking a look. Just to make sure I understand the issue you're describing, it's not about what happens after deleting a review - correct? It's about the fact that a number doesn't increase when adding a review?

If so that doesn't relate to this patch at all, and is likely a duplicate of #6120.

**Comment by ioanarusiczki** [comment]
@bobsilverberg Yes, I was talking about https://github.com/mozilla/addons-frontend/issues/6120
Thanks!
